VideoLocker/res/values/strings.xml

```diff
@@ -480,6 +480,8 @@
     <string name="login_error">Sign-in Error</string>
     <!-- Alert dialog title when login fails due to lack of internet -->
     <string name="no_connectivity">Connection Error</string>
+    <!-- When a download is about to start and we don't know it's file size yet -->
+    <string name="download_pending">Download pending…</string>
```

Comment:
Maybe group this together with `video_download_in_progress` and `msg_started_one_video_download`?